### PR TITLE
executeInSelenium action does not generate proper code

### PR DESCRIPTION
**Pull request by zakdma**
**executeInSelenium** action does not generate proper code because this action was not included in `ActionObject::FUNCTION_CLOSURE_ACTIONS`

### Steps to reproduce
1. Create test with **executeInSelenium** action 
`<executeInSelenium function="function ($webdriver) { return 'Hello, World!'}" stepKey="testStep"/>`
2. Run `vendor/bin/mftf generate:tests`

### Expected result
There is generated action
`$I->executeInSelenium(function ($webdriver) { return 'Hello, World!'});`
Function is passed as Closure.

### Actual result
There is generated action
`$I->executeInSelenium("function ($webdriver) { return 'Hello, World!'}");`
Function is passed as string not Closure. So if you run test you will get exception here.

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/verification tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)
 - [ ] Changes to Framework doesn't have backward incompatible changes for tests or have related Pull Request with fixes to tests

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/magento2-functional-testing-framework?pullRequest=332) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/magento2-functional-testing-framework?pullRequest=332) before we can accept your contribution.<br/>

**Comment by KevinBKozan** [comment]
Hey @zakdma, thank you for your contribution! We'll be processing your PR shortly, and I created an internal ticket to track progress.

File: dev/tests/verification/Tests/ExecuteInSeleniumTest.php [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+class ExecuteInSeleniumTest extends MftfTestCase
+{
+    /**
+     * Tests generation of executeInSelenium action.
+     *
+     * @throws \Exception
+     * @throws \Magento\FunctionalTestingFramework\Exceptions\TestReferenceException
+     */
+    public function testExecuteJsTest()
```
Comment (jilu1) [inline dev/tests/verification/Tests/ExecuteInSeleniumTest.php]: Possible renaming this to testExecuteInSeleniumTest()?

Comment (zakdma) [reply dev/tests/verification/Tests/ExecuteInSeleniumTest.php]: Yes of course

Comment (zakdma) [reply dev/tests/verification/Tests/ExecuteInSeleniumTest.php]: @jilu1 I have changed test name.
Please proceed with review

**Pull request merged**